OneCommunityGlobal/HGNRest
# Luis unit testing warnings controller.js

# Description
Unit testing for warnings controller 

## Related PRS (if any):
N/A

## Main changes explained:

- unit tests for the warnings controller. 
- Created tests for the getWarningsById method.
- Created tests for the postWarningsById method.
- Created tests for the deleteWarningsById method.

## How to test:

1. check into current branch
2. do `npm install`
3. run `npm run test warningsController`
4. verify all tests pass

## Screenshots or videos of changes:
<img width="1278" alt="Screenshot 2024-04-24 at 8 28 46 PM" src="https://github.com/OneCommunityGlobal/HGNRest/assets/22454953/7eb838d5-eb63-4f3b-93a5-c9eb043e892b">


## Note:
Include the information the reviewers need to know.
